pandas/tests/series/test_constructors.py

```diff
@@ -635,6 +635,10 @@ def test_fromValue(self):
         self.assertEqual(dates.dtype, 'M8[ns]')
         self.assertEqual(len(dates), len(self.ts))
 
+        categorical = Series(0, index=self.ts.index, dtype="category")
+        self.assertEqual(categorical.dtype, 'category')
```

Comment:
construct the expected series and use ``assert_series_equal`` (like you had before).